Library/Formula/clblas.rb

```diff
@@ -0,0 +1,23 @@
+class Clblas < Formula
+  homepage "https://github.com/clMathLibraries/clBLAS"
+  url "https://github.com/clMathLibraries/clBLAS/archive/v2.2.tar.gz"
+  sha256 "0563e028485d240f8b6e6efcae61677049627b19e924dcb1ef8014065b6416df"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+
+  patch do
+    # install clBLAS cmake configuration and version files
+    url "https://github.com/clMathLibraries/clBLAS/pull/68.diff"
+    sha1 "b85a17cd972ed598a80aef8623830c90f5abb1f4"
+  end
+
+  def install
+    system "cmake", "-DBUILD_TEST=OFF", "-DBUILD_CLIENT=ON", "-DBUILD_KTEST=OFF", "-DSUFFIX_LIB:STRING=", "-DCMAKE_MACOSX_RPATH:BOOL=ON", "src", *std_cmake_args
```

Comment:
Might be worth separating this into an array to make it more readable.

```ruby

args = std_cmake_args
args << "-DBUILD_TEST=OFF" << "-DBUILD_CLIENT=ON" << "-DBUILD_KTEST=OFF"
args << "-DSUFFIX_LIB:STRING=" << "-DCMAKE_MACOSX_RPATH:BOOL=ON" << "src"

system "cmake", *args
```

Or similar.